### ISSUE TITLE
Add 100ms to creature round timer to compensate for lag

### ISSUE THREAD

**Issue by stormhalter** [jkachhad/Stormhalter#694]
Average latency for players seems to be ~100ms, a flat 100-200 ms addition to creature round timers accross all facets would help smooth the game out for people. [Suggested by HealthySnack]

**Comment by SlysterLOK** [comment]
Agreed. A setting of 1.7 or so for mobs would help.

**Comment by yarrim** [comment]
I just spent an absurd amount of time trying to dodge Doom.Orc in Beryl. I felt like I got .6-.8 rounds for each mob round, losing a round to input and network latency every 3-4 rounds, but I had good ping and the in-game UI shows 30-is ms latency. The mobs have the advantage of instant decision making and instance input. It feel less than fair. We definitely had fun in Ruby when we had a 100% (-latency) advantage over mobs, and that was too much, but getting a little of an advantage to account for network and human reaction seems like a good gameplay decision.
